cfme/common/provider.py

```diff
@@ -71,7 +71,7 @@ def form(self):
                 ('google_service_account', Input('service_account')),
             ]
             tab_fields = {
-                "Default": [
+                ("Default", ('default_when_no_tabs', )): [
```

Comment:
@jteehan This is the architectural thing I was talking about. This sets a flag on the tab that if no tabs are there, then if accessing a field that is under this tab do nothing. And if accessing a different tab, cry.